linter.py

```diff
@@ -73,3 +82,52 @@ def ensure_plugin_installed(self) -> bool:
 		# the environment. Silently, do not notify and disturb the user!
 		self.notify_unassign()
 		raise PermanentError()
+
+def guess_cwd(view):
+	if view.file_name():
+		return os.path.dirname(view.file_name())
+	elif len(view.window().folders()):
+		return view.window().folders()[0]
```

Comment:
Is there really no Sublime API to get the `cwd`?

Reply:
Sublime let we code without creating a file or having multiple tabs with different locations, so the cwd becomes more tricky. Anyways, they should provide a simple way to get the `cwd` because this a common use case.
As I understood if we use `os.getcwd()` we get the `cwd` of Sublime installation.

With that in mind, previously I already followed gofmt:
https://github.com/noonat/sublime-gofmt/blob/master/gofmt.py#L48
And now I tried to research deeply and others are using very similar code.